core/src/main/scala/org/apache/spark/network/netty/SparkTransportConf.scala

```diff
@@ -55,4 +57,31 @@ object SparkTransportConf {
       }
     })
   }
+
+  /**
+    * Separate threads configuration of driver and executor
```

Comment:
It seems something wrong with the intention, maybe you can fix it by setting Code Style in IntelliJ?

Reply:
Hi Deegue,

thanks for your comment. But can you point out what is wrong in this place? If you compare this method with the existing method, "fromSparkconf", the indentation looks same.

thanks.

Reply:
Ok, I see the difference. let me fix it.